package.json

```diff
@@ -59,8 +60,8 @@
     "nsp": "^2.6.3",
     "pre-commit": "^1.2.2",
     "standard-version": "^4.0.0",
-    "webpack": "^2.3.2",
-    "webpack-defaults": "^1.0.1"
+    "webpack": "^3.0.0",
```

Comment:
I don't think this is the right action to take - it increases friction for a new person checking out the repo and just wanting to `npm install && npm test`. Secondly the default for npm install in v5 is to always save to package.json and package-lock.json - so just doing `npm install webpack` to develop locally is always going to update those files unless the user explicitly adds the `--no-save` flag which again causes confusion and decreases the likliehood of someone wanting to contribute (and will probably lead to a user accidentally committing the changes to a PR and having to have this discussion again).